[CI] DocsClientYamlTestSuiteIT test {yaml=reference/rest-api/usage/line_38} failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/2yze5jzpws2l2/tests/:docs:integTest/org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT/test%20%7Byaml=reference%2Frest-api%2Fusage%2Fline_38%7D

**Reproduction line:**
`./gradlew ':docs:integTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/rest-api/usage/line_38}" -Dtests.seed=B1298726AF21E894 -Dtests.locale=en-MT -Dtests.timezone=Asia/Tomsk -Druntime.java=17`

**Applicable branches:**
master

**Reproduces locally?:**
No

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT&tests.test=test%20%7Byaml%3Dreference/rest-api/usage/line_38%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [reference/rest-api/usage:10]: expected [2xx] status code but api [raw[method=GET path=_xpack/usage]] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"resource_not_found_exception","reason":"unable to find script [my-prod-tag-script] in cluster state","processor_type":"drop","stack_trace":"org.elasticsearch.ResourceNotFoundException: unable to find script [my-prod-tag-script] in cluster state\n\tat org.elasticsearch.script.ScriptService.getScriptFromClusterState(ScriptService.java:680)\n\tat org.elasticsearch.script.ScriptService.compile(ScriptService.java:595)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:77)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:58)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:595)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:547)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessorConfigs(ConfigurationUtils.java:467)\n\tat org.elasticsearch.ingest.Pipeline.create(Pipeline.java:82)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:272)\n\tat java.base/java.util.HashMap.forEach(HashMap.java:1421)\n\tat java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.pipelineIdsByModelIdsOrAliases(TransportGetTrainedModelsStatsAction.java:270)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$doExecute$4(TransportGetTrainedModelsStatsAction.java:133)\n\tat org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136)\n\tat org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31)\n\tat org.elasticsearch.client.internal.node.NodeClient.lambda$executeLocally$0(NodeClient.java:107)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:176)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:170)\n\tat org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:447)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponseAsync(TransportNodesAction.java:181)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponse(TransportNodesAction.java:156)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction$AsyncAction.lambda$finishHim$0(TransportNodesAction.java:291)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"}],"type":"exception","reason":"unexpected failure gathering pipeline information","caused_by":{"type":"resource_not_found_exception","reason":"unable to find script [my-prod-tag-script] in cluster state","processor_type":"drop","stack_trace":"org.elasticsearch.ResourceNotFoundException: unable to find script [my-prod-tag-script] in cluster state\n\tat org.elasticsearch.script.ScriptService.getScriptFromClusterState(ScriptService.java:680)\n\tat org.elasticsearch.script.ScriptService.compile(ScriptService.java:595)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:77)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:58)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:595)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:547)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessorConfigs(ConfigurationUtils.java:467)\n\tat org.elasticsearch.ingest.Pipeline.create(Pipeline.java:82)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:272)\n\tat java.base/java.util.HashMap.forEach(HashMap.java:1421)\n\tat java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.pipelineIdsByModelIdsOrAliases(TransportGetTrainedModelsStatsAction.java:270)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$doExecute$4(TransportGetTrainedModelsStatsAction.java:133)\n\tat org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136)\n\tat org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31)\n\tat org.elasticsearch.client.internal.node.NodeClient.lambda$executeLocally$0(NodeClient.java:107)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:176)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:170)\n\tat org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:447)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponseAsync(TransportNodesAction.java:181)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponse(TransportNodesAction.java:156)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction$AsyncAction.lambda$finishHim$0(TransportNodesAction.java:291)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"},"stack_trace":"org.elasticsearch.ElasticsearchException: unexpected failure gathering pipeline information\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:287)\n\tat java.base/java.util.HashMap.forEach(HashMap.java:1421)\n\tat java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.pipelineIdsByModelIdsOrAliases(TransportGetTrainedModelsStatsAction.java:270)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$doExecute$4(TransportGetTrainedModelsStatsAction.java:133)\n\tat org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136)\n\tat org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31)\n\tat org.elasticsearch.client.internal.node.NodeClient.lambda$executeLocally$0(NodeClient.java:107)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:176)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:170)\n\tat org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:447)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponseAsync(TransportNodesAction.java:181)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponse(TransportNodesAction.java:156)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction$AsyncAction.lambda$finishHim$0(TransportNodesAction.java:291)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\nCaused by: org.elasticsearch.ResourceNotFoundException: unable to find script [my-prod-tag-script] in cluster state\n\tat org.elasticsearch.script.ScriptService.getScriptFromClusterState(ScriptService.java:680)\n\tat org.elasticsearch.script.ScriptService.compile(ScriptService.java:595)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:77)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:58)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:595)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:547)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessorConfigs(ConfigurationUtils.java:467)\n\tat org.elasticsearch.ingest.Pipeline.create(Pipeline.java:82)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:272)\n\t... 17 more\n"},"status":500}]

  at __randomizedtesting.SeedInfo.seed([B1298726AF21E894:397DB8FC01DD856C]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:489)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:462)
  at jdk.internal.reflect.GeneratedMethodAccessor14.invoke(null:-1)
  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:568)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
  at java.lang.Thread.run(Thread.java:833)

  Caused by: java.lang.AssertionError: expected [2xx] status code but api [raw[method=GET path=_xpack/usage]] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"resource_not_found_exception","reason":"unable to find script [my-prod-tag-script] in cluster state","processor_type":"drop","stack_trace":"org.elasticsearch.ResourceNotFoundException: unable to find script [my-prod-tag-script] in cluster state\n\tat org.elasticsearch.script.ScriptService.getScriptFromClusterState(ScriptService.java:680)\n\tat org.elasticsearch.script.ScriptService.compile(ScriptService.java:595)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:77)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:58)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:595)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:547)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessorConfigs(ConfigurationUtils.java:467)\n\tat org.elasticsearch.ingest.Pipeline.create(Pipeline.java:82)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:272)\n\tat java.base/java.util.HashMap.forEach(HashMap.java:1421)\n\tat java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.pipelineIdsByModelIdsOrAliases(TransportGetTrainedModelsStatsAction.java:270)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$doExecute$4(TransportGetTrainedModelsStatsAction.java:133)\n\tat org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136)\n\tat org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31)\n\tat org.elasticsearch.client.internal.node.NodeClient.lambda$executeLocally$0(NodeClient.java:107)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:176)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:170)\n\tat org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:447)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponseAsync(TransportNodesAction.java:181)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponse(TransportNodesAction.java:156)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction$AsyncAction.lambda$finishHim$0(TransportNodesAction.java:291)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"}],"type":"exception","reason":"unexpected failure gathering pipeline information","caused_by":{"type":"resource_not_found_exception","reason":"unable to find script [my-prod-tag-script] in cluster state","processor_type":"drop","stack_trace":"org.elasticsearch.ResourceNotFoundException: unable to find script [my-prod-tag-script] in cluster state\n\tat org.elasticsearch.script.ScriptService.getScriptFromClusterState(ScriptService.java:680)\n\tat org.elasticsearch.script.ScriptService.compile(ScriptService.java:595)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:77)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:58)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:595)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:547)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessorConfigs(ConfigurationUtils.java:467)\n\tat org.elasticsearch.ingest.Pipeline.create(Pipeline.java:82)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:272)\n\tat java.base/java.util.HashMap.forEach(HashMap.java:1421)\n\tat java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.pipelineIdsByModelIdsOrAliases(TransportGetTrainedModelsStatsAction.java:270)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$doExecute$4(TransportGetTrainedModelsStatsAction.java:133)\n\tat org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136)\n\tat org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31)\n\tat org.elasticsearch.client.internal.node.NodeClient.lambda$executeLocally$0(NodeClient.java:107)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:176)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:170)\n\tat org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:447)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponseAsync(TransportNodesAction.java:181)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponse(TransportNodesAction.java:156)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction$AsyncAction.lambda$finishHim$0(TransportNodesAction.java:291)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"},"stack_trace":"org.elasticsearch.ElasticsearchException: unexpected failure gathering pipeline information\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:287)\n\tat java.base/java.util.HashMap.forEach(HashMap.java:1421)\n\tat java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.pipelineIdsByModelIdsOrAliases(TransportGetTrainedModelsStatsAction.java:270)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$doExecute$4(TransportGetTrainedModelsStatsAction.java:133)\n\tat org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:136)\n\tat org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:31)\n\tat org.elasticsearch.client.internal.node.NodeClient.lambda$executeLocally$0(NodeClient.java:107)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:176)\n\tat org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:170)\n\tat org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:447)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponseAsync(TransportNodesAction.java:181)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction.newResponse(TransportNodesAction.java:156)\n\tat org.elasticsearch.action.support.nodes.TransportNodesAction$AsyncAction.lambda$finishHim$0(TransportNodesAction.java:291)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:717)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\nCaused by: org.elasticsearch.ResourceNotFoundException: unable to find script [my-prod-tag-script] in cluster state\n\tat org.elasticsearch.script.ScriptService.getScriptFromClusterState(ScriptService.java:680)\n\tat org.elasticsearch.script.ScriptService.compile(ScriptService.java:595)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:77)\n\tat org.elasticsearch.ingest.ConditionalProcessor.<init>(ConditionalProcessor.java:58)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:595)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessor(ConfigurationUtils.java:547)\n\tat org.elasticsearch.ingest.ConfigurationUtils.readProcessorConfigs(ConfigurationUtils.java:467)\n\tat org.elasticsearch.ingest.Pipeline.create(Pipeline.java:82)\n\tat org.elasticsearch.xpack.ml.action.TransportGetTrainedModelsStatsAction.lambda$pipelineIdsByModelIdsOrAliases$16(TransportGetTrainedModelsStatsAction.java:272)\n\t... 17 more\n"},"status":500}]

    at org.junit.Assert.fail(Assert.java:88)
    at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:374)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:478)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:462)
    at jdk.internal.reflect.GeneratedMethodAccessor14.invoke(null:-1)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:568)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
    at java.lang.Thread.run(Thread.java:833)

```

--- a/docs/reference/ingest.asciidoc
+++ b/docs/reference/ingest.asciidoc
@@ -883,6 +883,7 @@ PUT _ingest/pipeline/my-pipeline
 [source,console]
 ----
 DELETE _scripts/my-prod-tag-script
+DELETE _ingest/pipeline/my-pipeline
 ----
 // TEST[continued]
 ////

